pkg/cli/admin/inspect/resource.go

```diff
@@ -274,15 +280,23 @@ func obtainRelatedObjects(obj *unstructured.Unstructured) ([]*configv1.ObjectRef
 		return nil, nil
 	}
 
+	errs := []error{}
```

Comment:
I don't think we would need any changes in this function.

Reply:
Our only goal is to write the yaml of object in cases of error is returned. The rest piece of code should stay as is. Am I missing something?

Reply:
Could you please point me the line of where `gatherRelatedObjects` function return error?

Reply:
Here: https://github.com/openshift/oc/blob/3544c64b893926687ae799475168d6389022c121/pkg/cli/admin/inspect/resource.go#L226

`gatherMoreObjects` has an error because the ObjectReference is empty, because the CR's status.relatedObjects are not real ObjectReferences

Reply:
I think, we can leave this as is. Same error (although it is not clear) returns with/without this patch.